kubernetes/schema_container.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
-func handlerFields() map[string]*schema.Schema {
+func lifecycleHandlerFields() map[string]*schema.Schema {
```

Comment:
In version `v0.23.5` they renamed the structure from `Handler` to `LifecycleHandler` of `spec.containers.lifecycle.[postStart|preStop]`.